bot/exts/moderation/infraction/infractions.py

```diff
@@ -363,6 +422,32 @@ async def pardon_ban(self, user_id: int, guild: discord.Guild, reason: t.Optiona
 
         return log_text
 
+    async def pardon_voice_ban(self, user_id: int, guild: discord.Guild, reason: t.Optional[str]) -> t.Dict[str, str]:
+        """Add Voice Verified role back to user, DM them a notification, and return a log dict."""
+        user = guild.get_member(user_id)
+        log_text = {}
+
+        if user:
+            # Add Voice Verified role back to user.
+            self.mod_log.ignore(Event.member_update, user.id)
+            await user.add_roles(self._voice_verified_role, reason=reason)
+
+            # DM user about infraction expiration
+            notified = await _utils.notify_pardon(
+                user=user,
+                title="Your Voice Ban have been removed",
+                content="You can now speak again in voice channels.",
+                icon_url=_utils.INFRACTION_ICONS["voice_ban"][1]
+            )
+
+            log_text["Member"] = format_user(user)
+            log_text["DM"] = "Sent" if notified else "**Failed**"
+        else:
+            log.info(f"Failed to remove Voice Ban from user {user_id}: user not found")
+            log_text["Failure"] = "User was not found in the guild."
```

Comment:
It isn't worth logging this as a failure, the infraction was pardoned.
```suggestion
            log_text["Info"] = "User was not found in the guild."
```